beacon-chain/p2p/custody.go

```diff
@@ -115,6 +115,59 @@ func (s *Service) UpdateCustodyInfo(earliestAvailableSlot primitives.Slot, custo
 	return earliestAvailableSlot, custodyGroupCount, nil
 }
 
+// UpdateEarliestAvailableSlot updates only the earliest available slot while preserving
+// the current custody group count. It returns the (possibly updated) earliest available slot
+// and the custody group count.
+//
+// IMPORTANT: This function should only be called when Fulu is enabled. The caller is responsible
+// for checking params.FuluEnabled() before calling this function.
+func (s *Service) UpdateEarliestAvailableSlot(earliestAvailableSlot primitives.Slot) (primitives.Slot, uint64, error) {
```

Comment:
The `cgc` output of this function is never used. I think we can remove it for now.